src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -127,6 +126,26 @@ def record_set(self, train, labels=None, channel="train"):
         logger.debug("Created manifest file {}".format(manifest_s3_file))
         return RecordSet(manifest_s3_file, num_records=train.shape[0], feature_dim=train.shape[1], channel=channel)
 
+    def record_set_from_local_files(self, data_path, num_records, feature_dim, channel="train"):
+        """Build a :class:`~RecordSet` by pointing to local files.
+
+        Args:
+            data_path (string): Path to local file to be uploaded for training.
```

Comment:
Need more specificity either here or in the overall comment. The user needs to know whether it works on single files, directories, or both, etc.